client/extensions/woocommerce/woocommerce-services/views/label-settings/label-payment-method.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormCheckbox from 'components/forms/form-checkbox';
+import CompactCard from 'components/card/compact';
+import PaymentLogo from 'components/payment-logo';
+
+const PaymentMethod = ( { translate, selected, isLoading, type, digits, name, expiry, onSelect } ) => {
+	const renderPlaceholder = () => (
+		<CompactCard className="label-settings__card">
+			<FormCheckbox
+				className="label-settings__card-checkbox" />
+			<PaymentLogo className="label-settings__card-logo" type="placeholder" />
+			<div className="label-settings__card-details">
+				<p className="label-settings__card-number" />
+				<p className="label-settings__card-name" />
+			</div>
+			<div className="label-settings__card-date">
+				<p />
+			</div>
+		</CompactCard>
+	);
+
+	if ( isLoading ) {
+		return renderPlaceholder();
+	}
+
+	const supportedTypes = {
+		amex: translate( 'American Express' ),
+		discover: translate( 'Discover' ),
+		mastercard: translate( 'MasterCard' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Visa, MasterCard' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).